_std/macros/list.dm

```diff
@@ -38,5 +38,7 @@
 #define LAZYLISTREMOVEASSOC(L, K, V) if(L) { if(L[K]) { L[K] -= V; if(!length(L[K])) L -= K; } if(!length(L)) L = null; }
 
 /// Accesses an associative list, returns null if nothing is found
-#define LAZYLISTACCESSASSOC(L, I, K) L ? L[I] ? L[I][K] ? L[I][K] : null : null : null
+#define LAZYLISTACCESSASSOC(L, K) (L ? L[K] : null)
```

Comment:
https://github.com/tgstation/tgstation/blob/923b4545d8f00c611c57c23e5f16f5d20ec4ec42/code/__HELPERS/_lists.dm#L84

this is what tg uses